src/frontend/src/hooks/projects.hooks.ts

```diff
@@ -58,3 +64,24 @@ export const useEditSingleProject = (wbsNum: WbsNumber) => {
     }
   );
 };
+
+/**
+ * Custom React Hook to set the team for a project
+ * @param wbsNum the project to set the team for
+ * @param teamId the id of the team to set the project to
+ */
+export const useSetProjectTeam = (wbsNum: WbsNumber) => {
+  const queryClient = useQueryClient();
+  return useMutation<{ message: string }, Error, any>(
+    ['projects', 'edit', 'teams'],
+    async (teamId: string) => {
+      const { data } = await setProjectTeam(wbsNum, teamId);
+      return data;
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries(['teams']);
```

Comment:
this will need to invalidate `projects` too so that the projects page is updated becasue that's where we'll be doing this from anyways